language/de-DE/joomla.ini

```
@@ -196,7 +196,7 @@ JFIELD_MEDIA_EMBED_CHECK_DESC_LABEL="Native Elemente audio, video oder object ve
 JFIELD_MEDIA_EMBED_CHECK_LABEL="Einbetten"
 JFIELD_MEDIA_CLASS_LABEL="CSS-Klasse Bild"
 JFIELD_MEDIA_FIGURE_CAPTION_LABEL="Bildunterschrift"
-JFIELD_MEDIA_FIGURE_CLASS_LABEL="CSS-Klasse Bildunterschrift"
+JFIELD_MEDIA_FIGURE_CLASS_LABEL="CSS-Klasse Bild-Container"
```

```suggestion
JFIELD_MEDIA_FIGURE_CLASS_LABEL="CSS-Klasse Abbildung"
```
Das wäre etwas allgemeiner. Auch OK? @dozentis ?

---

Ich hatte tatsächlich "Abbildung" auch als 2. mögliche Option geschrieben, dann diese rausgenommen und mich auf Bild-Container festgelegt, da ich finde dass in dieser Maske damit der Zweck des Feldes besser verstanden werden kann, als Abbildung (was natürlich die sinngemäße Übersetzung ist)

 [https://www.mediaevent.de/html/figure-figcaption.html](url)
> HTML figure ist ein Container für Bilder, Grafiken und andere Abbildungen

---

Nochmal nachgedacht: Da über CMS-Inhalte/Medien einfügen auch neben Bilder, auch audio/video/object  Elemente geladen werden können, wäre vielleicht sogar "CSS Klasse Medien-Container" passender? 🤔